tests/bootstrap.php

```diff
@@ -5,4 +5,26 @@
  */
 
 $loader = require __DIR__ . "/../vendor/autoload.php";
-$loader->addPsr4( 'YOURLS\\', __DIR__ );
+
+
+/**
+* Returns the name of a temporary file
+*/
+function temp_filename() {
+    $tmp_dir = realpath(__DIR__);
+    return tempnam($tmp_dir, 'pomo');
+}
+
+function replace_r_n($text) {
+    return str_replace("\r\n", "\n", $text);
+}
+
+// PHPUnit 6 compatibility for previous versions
+if (class_exists('PHPUnit\Runner\Version' ) && version_compare( PHPUnit\Runner\Version::id(), '6.0', '>=' ) ) {
```

Comment:
Needed to run latest PHPUnit available on each PHP version: PHPUnit 4.8 on PHP 5.x, PHPUnit 6 on PHP 7, which is not backward compatible.

Alternative would be to force PHPUnit 4.8 on each version of PHP which I find is less desirable.